documentation/user_docs/docs/manual/Tobyfit.rst

```diff
@@ -21,7 +21,7 @@ Resolution effects can be quite considerable. Not only do they result in an incr
 
 Tobyfit uses the multifit fitting interface to enable you fit your data to a model for S(Q,w) together with background functions. By default the S(Q,w) model can be global to all of the datasets you pass to Tobyfit i.e. a single model that applies with the same parameter values to all datasets, with independent background functions for each of the datasets. Alternatively, you can specify that the S(Q,w) model is local i.e. the parameters vary independently for each dataset, but you can constrain parameters to be bound between the S(Q,w) models so that, for example, they all share common magnetic exchange constants but the intensities and intrinsic lifetimes can vary independently for each cut.
 
-Tobyfit therefore appears to operate just like :ref:`multifit_sqw <Multifit:multifit_sqw>`, with the same set of capabilities of controlling parameter setting, binding etc that all other :ref:`multifit variants <Multifit:Multifit>` have. The difference is that Tobyfit uses instrument information in the sqw objects to convolute the S(Q,w) model(s) with the instrument resolution function using a Monte Carlo multi-dimensional integration, and provides some additional methods to control how the convolution is carried out. Note that the background functions are \*not\* convoluted with the resolution function - the assumption is that they are simply empirical functions such as linear background models and so resolution function convolution makes little sense.
+Tobyfit therefore appears to operate just like :ref:`multifit_sqw <manual/Multifit:multifit_sqw>`, with the same set of capabilities of controlling parameter setting, binding etc that all other :ref:`multifit variants <manual/Multifit:Multifit>` have. The difference is that Tobyfit uses instrument information in the sqw objects to convolute the S(Q,w) model(s) with the instrument resolution function using a Monte Carlo multi-dimensional integration, and provides some additional methods to control how the convolution is carried out. Note that the background functions are \*not\* convoluted with the resolution function - the assumption is that they are simply empirical functions such as linear background models and so resolution function convolution makes little sense.
```

Comment:
An issue we are addressing in the sqw redesign is that instrument data may not have been available in the construction of the original sqw, and may have been added immediately before doing a Tobyfit. Documenting that process here would be outside scope, but mentioning that the user should make sure the instrument info is in place probably should be.

Reply:
I've added a sentence after this noting that in future `gen_sqw` will include instrument information in a `sqw` file if the input files support it.